rascal2/ui/presenter.py

```diff
@@ -32,9 +32,45 @@ def create_project(self, name: str, save_path: str):
             The save path of the project.
 
         """
+        self.model.create_project(name, save_path)
+        self.initialise_ui(name, save_path)
+
+    def load_project(self, load_path: str):
+        """Load an existing RAT project then initialise UI.
+
+        Parameters
+        ----------
+        load_path : str
+            The path from which to load the project.
+
+        """
+        self.model.load_project(load_path)
```

Comment:
in any case i'm dealing with it in the `verify_folder` methods of the dialog so that the error can be displayed in the project dialog - then all the folder verification happens in one place